src/node_http_parser.cc

```diff
@@ -136,7 +136,10 @@ struct StringPtr {
 
   Local<String> ToString(Environment* env) const {
     if (str_)
-      return OneByteString(env->isolate(), str_, size_);
+      return String::NewFromUtf8(env->isolate(),
```

Comment:
@ronkorving Yes. v0.10 reads in headers as utf-8. Which is incorrect per the specification.

Thing is, if everyone followed US-ASCII then functionally we wouldn't notice a difference. It's not until someone uses the full latin-1 encoding (which is spec compliant) that they'd experience issues.